src/main/java/com/legobmw99/allomancy/modules/powers/CommonEventHandler.java

```diff
@@ -45,6 +45,21 @@ public static void onAttachCapability(final AttachCapabilitiesEvent<Entity> even
             event.addCapability(AllomancerCapability.IDENTIFIER, provider);
         }
     }
+    
+    public static int randomNumbGen(Player player) {
```

Comment:
I would prefer if this logic all just got moved inside the conditional. It doesn't get reused anywhere, so I don't think it needs to be a function